editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -310,19 +310,48 @@ fn line_join_widget(document_node: &DocumentNode, node_id: u64, index: usize, na
 	LayoutGroup::Row { widgets }
 }
 
+fn fill_type_widget(document_node: &DocumentNode, node_id: u64, index: usize, name: &str, blank_assist: bool) -> LayoutGroup {
```

Comment:
Let's remove the `name` and `blank_assist` parameters from this function signature, and just hard-code their values in the line below in place of their variables. So in other words, move the hard-coded values from the caller of this function to the `let mut widgets = start_widgets(...);` line.